### PR TITLE
runit, runsv, runsvdir, runsvchdir, sv: add more information link and move to common/

### PR DESCRIPTION
**Pull request by marchersimon**
For #5510

### CODE REVIEW

**Comment by marchersimon** [comment]
Actually there are some other pages in the runit bundle, which are in the wrong directory. Should I add them here?

**Comment by bl-ue** [comment]
Sure, just add their names to the PR title e.g. `runit, page1, page: ...`

**Comment by marchersimon** [comment]
Just found some more

**Comment by bl-ue** [comment]
Well, changes like this are what keep the older parts of tldr-pages from decaying and crumbling away.  It's always good to refresh all the pages every once in a while 👍🏻

File: pages.zh/common/runsvchdir.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# runsvchdir
+
+> 更改默认使用的 `runsvdir` 目录.
+> 更多信息： <https://manpages.ubuntu.com/manpages/latest/en/man8/runsvchdir.8.html>.
```
Comment (bl-ue) [inline pages.zh/common/runsvchdir.md:4]:
```suggestion
> 更多信息： <https://manpages.ubuntu.com/manpages/latest/man8/runsvchdir.8.html>.
```

File: pages/common/runsvchdir.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # runsvchdir
 
 > Change the directory `runsvdir` uses by default.
+> More information: <https://manpages.ubuntu.com/manpages/latest/en/man8/runsvchdir.8.html>.
```
Comment (bl-ue) [inline pages/common/runsvchdir.md:4]:
```suggestion
> More information: <https://manpages.ubuntu.com/manpages/latest/man8/runsvchdir.8.html>.
```

File: pages.zh/common/runsvdir.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 # runsvdir
 
 > 运行整个目录下的服务.
+> 更多信息： <https://manpages.ubuntu.com/manpages/latest/en/man8/runsvdir.8.html>.
```
Comment (bl-ue) [inline pages.zh/common/runsvdir.md:4]:
```suggestion
> 更多信息： <https://manpages.ubuntu.com/manpages/latest/man8/runsvdir.8.html>.
```

File: pages/common/runsvdir.md [hunk 4]
```diff
@@ -1,6 +1,7 @@
 # runsvdir
 
 > Run an entire directory of services.
+> More information: <https://manpages.ubuntu.com/manpages/latest/en/man8/runsvdir.8.html>.
```
Comment (bl-ue) [inline pages/common/runsvdir.md:4]:
```suggestion
> More information: <https://manpages.ubuntu.com/manpages/latest/man8/runsvdir.8.html>.
```

**Review by bl-ue — CHANGES_REQUESTED** [review]
Remove `/en` i18n

**Comment by marchersimon** [comment]
Well, after that PR, I have successfully eliminated over 50% of all sunos pages :sweat_smile:

**Comment by marchersimon** [comment]
What sould we do about [pages.zh/sunos/sv.md](https://github.com/tldr-pages/tldr/blob/master/pages.zh/sunos/sv.md) and [pages.zh/osx/sv.md](https://github.com/tldr-pages/tldr/blob/master/pages.zh/osx/sv.md)?

**Comment by bl-ue** [comment]
If you eliminate all of them, we can remove the whole platform, and then re-add it some time as `solaris` 🤓

**Comment by marchersimon** [comment]
> If you eliminate all of them, we can remove the whole platform, and then re-add it some time as `solaris` nerd_face

There would be only 4 pages left in sunos and I don't think there'll be new ones

**Comment by bl-ue** [comment]
>What sould we do about [pages.zh/sunos/sv.md](https://github.com/tldr-pages/tldr/blob/master/pages.zh/sunos/sv.md) and [pages.zh/osx/sv.md](https://github.com/tldr-pages/tldr/blob/master/pages.zh/osx/sv.md)?

Discard the latter. The former is more recent ([Oct 2020](https://github.com/tldr-pages/tldr/pull/4389) vs. [March 2019](https://github.com/tldr-pages/tldr/pull/2831)) and it's slightly more accurate (the variables in the latter are e.g. `target directory/target service` vs. `directory/service` etc.)

**Comment by bl-ue** [comment]
>There would be only 4 pages left in sunos and I don't think there'll be new ones

Maybe we could figure out that they are usable on Linux and move them there (or common/)

**Review by bl-ue — APPROVED** [review]
Love these big cleanup PRs! 🤺

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**